### PR TITLE
Added the ability to create a database with an environment variable.

### PR DESCRIPTION
**Pull request by Tebro**
If a user wants to create a database when the container is created all they have to
do is supply the MYSQL_DB environment variable.

### CODE REVIEW

**Comment by tifayuki** [comment]
The name of `MYSQL_DB` is not that intuitive, I think.
In our influxdb image, I named it as `PRE_CREATE_DB`. Can you change `MYSQL_DB` to something similar?

Thank you

**Comment by Tebro** [comment]
Of course I can, if we can agree on the name.

Can also be: ON_CREATE_DB, or FIRST_DB or something similar. If a collaborator decides the name and posts a definitive answer on here I will change it.

**Comment by tifayuki** [comment]
Let's  do `ON_CREATE_DB`

**Comment by Tebro** [comment]
There. Took 2 push --force since I forgot to git add first..

**Comment by Tebro** [comment]
The CI gives me a 404 so I cannot check what is failing.

**Comment by tifayuki** [comment]
I re-triggered the test and it passed.

Thank you very much :)

**Comment by nafg** [comment]
@Tebro see above

File: 5.5/Dockerfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ RUN chmod 755 /*.sh
 # Exposed ENV
 ENV MYSQL_USER admin
 ENV MYSQL_PASS **Random**
+ENV ON_CREATE_DB **False**
```
Comment (nafg) [inline 5.5/Dockerfile]: What's the purpose of setting a value that flags it to later unset it? Why not leave it undefined by default?

**Comment by Tebro** [comment]
Ah you are right on both parts. Will look into producing a fix today.

**Comment by nafg** [comment]
@Tebro sure but if you want the default unset, you don't need an ENV line for it.

If you just want to document its existence, you could include a comment like
```
# ENV ON_CREATE_DB cd_library
```

**Pull request merged**